cosmosdb/sql/ps-container-create-index-none.ps1

```diff
@@ -19,6 +58,6 @@ $ContainerProperties = @{
     "options"=@{ "Throughput"="400" }
 } 
 
-New-AzResource -ResourceType "Microsoft.DocumentDb/databaseAccounts/apis/databases/containers" `
-    -ApiVersion "2015-04-08" -ResourceGroupName $resourceGroupName `
+New-AzResource -ResourceType $containerResourceType `
+    -ApiVersion $apiVersion -ResourceGroupName $resourceGroupName `
     -Name $resourceName -PropertyObject $ContainerProperties
```

Comment:
`$resourceName` is used here, but never defined in the script.

Reply:
Fixed in next commit.